mne/io/base.py

```diff
@@ -1424,7 +1424,7 @@ def save(self, fname, picks=None, tmin=0, tmax=None, buffer_size_sec=None,
         check_fname(fname, 'raw', endings, endings_err=endings_err)
 
         split_size = _get_split_size(split_size)
-        if not self.preload and fname in self._filenames:
+        if not self.preload and str(fname) in self._filenames:
```

Comment:
Nowadays we don't cast using `str` but rather use `fname = _check_fname(fname)` because it does some nice standardization for us

Reply:
Done.